### PR TITLE
Add GetPayment method

### PR DESCRIPTION
**Pull request by dennisreimann**
This adds the `GetPayment` method, which allows to retrieve the payment info by payment hash.

### CODE REVIEW

**Comment by dennisreimann** [comment]
Let's bring in #66 first, I'll then rebase this one.

File: src/BTCPayServer.Lightning.LND/Base64Url.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        }
+
+        private static string ToBase64UrlString(byte[] arg)
+        {
+            if (arg == null)
+            {
+                throw new ArgumentNullException("arg");
+            }
+
+            var s = Convert.ToBase64String(arg);
```
Comment (Kukks) [inline src/BTCPayServer.Lightning.LND/Base64Url.cs:19]: you can use ` Encoders.Base64.EncodeData(arg)`?

File: src/BTCPayServer.Lightning.LND/Base64Url.cs [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            return s
+                .Replace("/", "_")
+                .Replace("+", "-");
+        }
+
+        private static byte[] HexStringToHex(this string inputHex)
+        {
```
Comment (Kukks) [inline src/BTCPayServer.Lightning.LND/Base64Url.cs:26]: You can just do `Encoders.Hex.DecodeData(inputHex)`

**Review by Kukks — CHANGES_REQUESTED** [review]

File: src/BTCPayServer.Lightning.LND/Base64Url.cs (hunk 1 above)
Comment (dennisreimann) [reply src/BTCPayServer.Lightning.LND/Base64Url.cs]: Thanks, done in [823ba0f](https://github.com/btcpayserver/BTCPayServer.Lightning/pull/64/commits/823ba0f4134a40ca7d2cb6f8b81e565b17f4157e).

**Pull request merged**